frontend/src/Pages/App/App.css

```diff
@@ -0,0 +1,4 @@
+.App{
+  width: 100vw;
+  height: 100vh;
+}
```

Comment:
This looks correct code-wise and syntax but for some reason, the linter is marking this off.. maybe it could be the style sheet that the linter is referencing.